server/controllers/users.js

```diff
@@ -138,20 +159,137 @@ class UserController {
             model: models.Profile,
             as: 'profile',
             attributes: ['firstname', 'lastname', 'bio', 'avatar', 'location']
-          },
-        ],
+          }
+        ]
       });
 
-      return res.status(200)
-        .send({
-          status: 'success',
-          message: 'Users and corresponding profiles retrieved successfully',
-          payload: users
-        });
+      return res.status(200).send({
+        status: 'success',
+        message: 'Users and corresponding profiles retrieved successfully',
+        payload: users
+      });
     } catch (error) {
       next(error);
     }
   }
+
+  /**
+   * Forgot Password
+   * @async
+   * @param  {object} req - Request object
+   * @param {object} res - Response object
+   * @param {object} next
+   * @return {json} Returns json object
+   * @static
+   */
+  static async forgotPassword(req, res, next) {
+    try {
+      const { email } = await validateForgotPassword(req.body);
+
+      const user = await User.findOne({ where: { email } });
+
+      if (!user) return Response.error(res, 404, 'User does not exist');
+
+      // Check if the account is active
+      if (!user.active) return Response.error(res, 403, 'Your account is not verified');
+
+      const token = await UserController.updateToken(user);
+
+      await sendForgotPasswordMail(token, email, user.get().username);
```

Comment:
no need awaiting this.

Let it resolve in the background

Reply:
I attempted to do that but it caused the tests for that feature to fail because the structure of the response was different from the structure of response from the Response.success()

Since I was not the one that worked on that ticket I communicated with @adex001 and he said it was fine that I can go ahead and resolve it. 

I changed the ones that did not affect the test tho.

Reply:
You are telling me
```suggestion
     sendForgotPasswordMail(token, email, user.get().username);
```
gives you issues?

Reply:
No this works fine. I have already implemented it. 

I was referring to the feedback from @adex, but I have also implemented that too. Everything works.